reflex-dom-canvas.cabal

```diff
@@ -27,19 +27,18 @@ library
                      , Reflex.Dom.Canvas.WebGL
                      , Reflex.Dom.Canvas.Context2D
 
-  build-depends:       base >=4.9 && <=4.12
+  build-depends:       base
```

Comment:
Please don't remove the upper bounds like this; you can use [this wiki page](https://wiki.haskell.org/Base_package) to get the correct upper bound.